app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-search-tracker.js

```diff
@@ -60,6 +60,11 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         // having seen the autocomplete and not used it, and not having seen it at all)
         if (this.$searchInput.dataset.autocompleteAccepted === 'true') {
           data.tool_name = 'autocomplete'
+
+          // Keep track of the fact that the autocomplete was accepted, so that we can use this as
+          // part of the `page_view` event on the subsequent page load
+          window.sessionStorage &&
+            window.sessionStorage.setItem('searchAutocompleteAccepted', 'true')
```

Comment:
I'm sure you're aware, but local storage comes under the umbrella of cookie consent, so we need to be careful with this stuff. I think this one is probably fine, but I wanted to mention it.

Reply:
Absolutely 💯 – both the reading and writing happen in module functions that won't run if the module hasn't been started, which it only is if consent has been given!

Reply:
Ah of course, sorry. Thanks for clarifying 👍